internal/cli/register.go

```diff
@@ -4,16 +4,50 @@ import (
 	"context"
 	"fmt"
 	"io/ioutil"
+	"os"
 
 	nodepair "github.com/mudler/go-nodepair"
 	qr "github.com/mudler/go-nodepair/qrcode"
 )
 
+// isDirectory determines if a file represented
+// by `path` is a directory or not
+func isDirectory(path string) (bool, error) {
```

Comment:
these two functions look quite generic, might be worth putting on a different package to be used by multiple commands from the CLI ... maybe even shared across project, not sure what't the architectural goal there